lib/utils/utilities.dart

```diff
@@ -12,11 +12,11 @@ String getInitials(String string, [int limitTo = 2]) {
 
   //For one word
   if (split.length == 1) {
-    return string.substring(0, 1);
+    return string.substring(0, 1).toUpperCase();
```

Comment:
@sriLatha2328  we don't need this upper casing as we are doing at the end.

Reply:
Here itself only we are returning the function. so, here i called toUpperCase() method .